static/styles/recent_topics.scss

```diff
@@ -66,6 +66,14 @@
 
         .recent_topics_focusable {
             display: inline-block;
+
+            & > * {
+                outline: 0px;
+            }
+
+            &:focus-within {
+                box-shadow: 0 3px 0 hsl(172, 100%, 42%);
```

Comment:
@timabbott You probably should adjust this color to what you feel is right, I think box-shadow is the correct way to do this.

Reply:
Yeah, this works.  I went with the color we use for the pointer in the main view, since those have the same role; I think it looks a bit worse than what you had here, but consistency seems better than anything else.

Might be worth thinking about changing the color for both of them, perhaps after extracting a CSS variable for the color :)